<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balance</class>
 <widget class="QDialog" name="balance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Fname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fname</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Lname</string>
   </property>
  </widget>
  <widget class="QLabel" name="accId">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>acc_id</string>
   </property>
  </widget>
  <widget class="QLabel" name="history">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>history</string>
   </property>
  </widget>
  <widget class="QLabel" name="accBalance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closebtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>243</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLabel" name="accType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
